syntax = "proto3";

option go_package = "pb/game";

message BeKillEvent 
{
  uint64 caster = 1;
  uint64 target = 2;
}
